packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -119,7 +120,7 @@ const BedAdministrationForm: React.FC<BedAdministrationFormProps> = ({
     <ComposedModal open={showModal} onClose={() => onModalChange(false)} preventCloseOnClickOutside>
       <ModalHeader title={headerTitle} />
       <Form onSubmit={handleSubmit(onSubmit, onError)}>
-        <ModalBody hasScrollingContent>
```

Comment:
Hi @IamMujuziMoses @chibongho, 
We should also pass `size` along with the `hasScrollingContent` prop for the scrolling to work, please test it out.
Thanks!